tests/integration/targets/role_member/tasks/main.yml

```
@@ -0,0 +1,134 @@
+---
+- name: Var block
+  vars:
+    login_name: "MrRoleTest"
+    plain_password: "P0pS3cret!23$%"
+    password_expiration_enabled: false
+    password_policy_enforced: false
+    password_must_change: false
+    enabled: false
+    default_database: "master"
+    language: "us_english"
+    default_schema: "dbo"
+    username: "MrRoleTest"
+    database: "master"
+  module_defaults:
+    lowlydba.sqlserver.role_member:
+      sql_instance: "{{ sqlserver_instance }}"
+      sql_username: "{{ sqlserver_username }}"
+      sql_password: "{{ sqlserver_password }}"
+      username: "{{ username }}"
+      database: "{{ database }}"
+    lowlydba.sqlserver.login:
+      sql_instance: "{{ sqlserver_instance }}"
+      sql_username: "{{ sqlserver_username }}"
+      sql_password: "{{ sqlserver_password }}"
+      default_database: "{{ default_database }}"
+      login: "{{ login_name }}"
+      password: "{{ plain_password }}"
+      password_expiration_enabled: "{{ password_expiration_enabled }}"
+      password_must_change: "{{ password_must_change }}"
+      enabled: "{{ enabled }}"
+      language: "{{ language }}"
+      state: present
+    lowlydba.sqlserver.user:
+      sql_instance: "{{ sqlserver_instance }}"
+      sql_username: "{{ sqlserver_username }}"
+      sql_password: "{{ sqlserver_password }}"
+      database: "{{ database }}"
+      login: "{{ login_name }}"
+      username: "{{ username }}"
+      default_schema: "{{ default_schema }}"
+      state: present
+  tags: ["role_member"]
+  block:
+    - name: Create login
+      lowlydba.sqlserver.login:
+      register: result
+    - assert:
+        that:
+          - result.data != None
+
+    - name: Create user
+      lowlydba.sqlserver.user:
+      register: result
+    - assert:
+        that:
+          - result.data != None
+
+    - name: Add roles for the user
+      lowlydba.sqlserver.role_member:
+        roles: db_datareader, db_datawriter
+      register: result
+    - assert:
+        that:
+          - result is changed
```

```suggestion
    - assert:
        that:
          - result is changed
          - result.data.SqlInstance != None
          - result.data.Database == "{{ database }}"
          - result.data.UserName == "{{ username }}"
```
Can you add assertions for some of the returned fields on each of the role tests? It should definitely check for the Role attribute as well but not sure off hand what the return format looks like